packages/grid/_modules_/grid/utils/domUtils.ts

```diff
@@ -74,3 +74,21 @@ export function getCellElementFromIndexes(
     `:scope .${CELL_CSS_CLASS}[aria-colIndex='${colIndex}'][data-rowIndex='${rowIndex}']`,
   ) as HTMLDivElement;
 }
+
+// Todo: Use the @material-ui/utils version once fixed
+// A change of the browser zoom change the scrollbar size.
+// Credit https://github.com/twbs/bootstrap/blob/3ffe3a5d82f6f561b82ff78d82b32a7d14aed558/js/src/modal.js#L512-L519
+export function getScrollbarSize(doc: Document): number {
```

Comment:
How about we move it with the other temporary patches?

Reply:
e.g. 

https://github.com/mui-org/material-ui-x/blob/e713ad39f3928f6c11acfe1b2406c6befc6e7d77/packages/grid/_modules_/grid/utils/material-ui-utils.ts#L11